.gitignore

```diff
@@ -0,0 +1,8 @@
+
+
+Starwar/Library/
+Starwar/Logs/
+Starwar/Temp/
+Starwar/UserSettings/
+Starwar/Packages/
```

Comment:
Remove this line to include Packages to the git. 
Last we tried, without the Packages folder, other cannot create .sln.